src/app/clusters/door-lock-server/door-lock-server.cpp

```diff
@@ -3812,6 +3854,110 @@ bool emberAfDoorLockClusterClearHolidayScheduleCallback(
     return true;
 }
 
+bool emberAfDoorLockClusterSetAliroReaderConfigCallback(
+    chip::app::CommandHandler * commandObj, const chip::app::ConcreteCommandPath & commandPath,
+    const chip::app::Clusters::DoorLock::Commands::SetAliroReaderConfig::DecodableType & commandData)
```

Comment:
Why all the extra prefixes?  Please remove the redundant prefixes.  Same elsewhere below.

Reply:
ah i though I checked this. will fix throughout

Reply:
done.